LICENSE

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2017, conda-forge
-All rights reserved.
```

Comment:
Yeah, the plan is to replace this PR with that one. Just wanted you to be aware of this issue for future submissions.